src/main/java/com/elixr/poc/Repository/PurchaseRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.elixr.poc.Repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface PurchaseRepository extends MongoRepository<com.elixr.poc.data.Purchase, UUID> {
```

Comment:
Why the purchase class is imported with package?

Reply:
I removed now.